src/prediction_market/related_models/prediction_market_vote_model.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+
+from utils.models import DefaultModel
+
+
+class PredictionMarketVote(DefaultModel):
+    created_by = models.ForeignKey(
+        "user.User",
+        related_name="prediction_market_votes",
+        blank=False,
+        null=False,
+        on_delete=models.CASCADE,
+    )
+
+    prediction_market = models.ForeignKey(
+        "prediction_market.PredictionMarket",
+        related_name="prediction_market_votes",
+        blank=False,
+        null=False,
+        on_delete=models.CASCADE,
+    )
+
+    vote = models.BooleanField(
```

Comment:
Linking back to a comment below, maybe this should be an enum instead of a True / False boolean so we can capture the state of 

"yes", "no", "neutral" (removed vote)